package/SUSEConnect.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon May  7 15:47:50 UTC 2018 - hschmidt@suse.com
+
+- Prevent the automatic registration of recommended products that
+  are not mirrored by the registration proxy.
+
```

Comment:
A link to the PR is probably needed for the submission request to be accepted.

Reply:
Really? I didn't know that. That could also be added to the Dangerfile:
Complain if your new changelog entry doesn't have a link to a PR, or a BSC, or FATE.

Reply:
I think it was because the bug was not marked as "resolved" or something like that?

Reply:
Nah, that was for SMT. :D